test/api/projects/instance-get/expected-state.json

```diff
@@ -15,6 +15,12 @@
           "name": "shopify",
           "description": "test fixture",
           "configuration": {
+            "authorization": {
+              "oauth_api": {
+                "id": "1234",
+                "version": 3
+              }
+            },
```

Comment:
Ta fixture (`internal/pkg/fixtures/configs/from-template.json`) se používá i v dalších testech, tak jsem je musel upravit všechny.

Reply:
`expected-state.json` je stav projektu po zavolání requestů. `GET` na `instanci` nic v konfiguraci nezmění, takže na konci je v ní to samý co tam na začátku vložila ta fixture.